torchtune/data/_utils.py

```diff
@@ -33,43 +31,30 @@ def truncate(
     return tokens_truncated
 
 
-def validate_messages(
-    messages: List[Message],
-) -> None:
+def split_text_by_image_tag(content: str, image_tag: str) -> List[Dict[str, str]]:
     """
-    Given a list of messages, ensure that messages form a valid
-    back-and-forth conversation. An error will be raised if:
-
-    - There is a system message that's not the first message
-    - There are two consecutive user messages
-    - An assistant message comes before the first user message
-    - The message is empty
-    - Messages are shorter than length of 2 (min. one user-assistant turn)
-
+    Given a raw text string, split by the specified ``image_tag``
+    and form into list of dictionaries to be used in the ``Message`` content
+    field.
```

Comment:
Can you be a bit more explicit about the format? E.g. at least "image tags will be represented as a dictionary {"type": "image"}"